wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -310,7 +310,7 @@ public final class SessionManager: NSObject, SessionManagerType {
 
     private let minTLSVersion: String?
 
-    private var analyticsManager: (any AnalyticsManagerProtocol)?
+    var analyticsManager: (any AnalyticsManagerProtocol)?
```

Comment:
I think this can be `private(set) var analyticsManager: (any AnalyticsManagerProtocol)?` as it is not `set` from outside

Reply:
Done: https://github.com/wireapp/wire-ios/commit/3946e2aeb8cd51f158b993d8cb73c180c15feb38